models/issue_content_history.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
Since we have a new `models` structure and this is a new file introduced. `models/issues/content_history.go` maybe good place for this file.

Reply:
Will wait for the avatar refactor

* https://github.com/go-gitea/gitea/pull/17123